docs/technical-reference.md

```diff
@@ -420,6 +453,31 @@ similar to the following to the KMS key policy:
 }
 ```
 
+#### 3. Add KMS Key Access to Quilt Role
+
+Finally, you need create a restricted policy that gives a Quilt role access to the keys for specific buckets, e.g:
+
+```json
+{
+  "Version": "2012-10-17",
+  "Statement": {
+    "Effect": "Allow",
+    "Action": [
+      "kms:Encrypt",
+      "kms:Decrypt"
+    ],
+    "Resource": [
+      "arn:aws:kms:us-west-2:111122223333:key/1234abcd-12ab-34cd-56ef-1234567890ab",
+      "arn:aws:kms:us-west-2:111122223333:key/0987dcba-09fe-87dc-65ba-ab0987654321"
+    ]
+  }
+}
+```
+
+You can now create a Quilt Policy from this policy (using the Catalog's admin panel, or `quilt3.admin.create_policy`).
```

Comment:
There is no `quilt3.admin.create_policy()` and actually `quilt3.admin` is broken needs to be fixed or removed. There is conversion about that somewhere in Slack and maybe ticket in Asana.

Reply:
Ouch. Okay, I will just remove those references and ensure there's an Asana ticket.

Reply:
Created new Task, assigned for Rob to prioritize https://app.asana.com/0/1201334960547652/1202896362990186/f

Reply:
Removed.